deployments/charts/kuma/templates/cp-webhooks-and-secrets.yaml

```diff
@@ -1,7 +1,16 @@
-{{- if not (eq (empty .Values.controlPlane.tls.general.caBundle) (empty .Values.controlPlane.tls.general.secretName)) }}
-  {{ fail "You need to send both or neither of controlPlane.tls.general.caBundle and controlPlane.tls.general.secretName"}}
-{{- end }}
-{{- $caBundle := .Values.controlPlane.tls.general.caBundle }}
+{{- $caBundle := .Values.controlPlane.tls.general.caBundle -}}
+{{- if (and (empty $caBundle) (ne .Values.controlPlane.tls.general.caSecretName "")) -}}
+  {{- if .Values.testOnly -}}
```

Comment:
I'm guessing an alternative is setting up real k8s cluster right? There is no other way?